### ISSUE TITLE
Module seems to be hiding/detaching more than requested.

### ISSUE THREAD

**Issue by bagarwa** [j-hc/zygisk-detach#32]
**Are you sure you read the [readme](https://github.com/j-hc/zygisk-detach#usage)?**
Yes

**Android version and skin (MIUI etc... is MIUI still a thing? idk):**
Android 14 QPR3 Beta 1

**Screenshot of Play Store page of the app you are trying to detach:**
![PlayStore](https://github.com/j-hc/zygisk-detach/assets/4424058/17659abb-cb61-437d-a8ea-105d35ce0bed)

**Screenshot of the helper CLI showing detached apps (from Option 2):**
![Termux](https://github.com/j-hc/zygisk-detach/assets/4424058/51cf0e19-cf1c-45cd-9220-4a5ce24fb45c)

**Logs taken with `su -c logcat -d -s "zygisk-detach" > /sdcard/log.txt`:**
Getting a 0B empty log.txt there after executing that command. 🤷‍♂️

# Issue
Described the issue in the first screenshot collage above. But will describe here again. 

Whenever I receive any notification from Play Store for pending updates, it shows a high number of pending updates. For example, the first part in the collage shows 23 pending updates. But when I tap the notification to reach play store app update section, it always shows very low number of app updates. (2 apps in the screenshot.) After I finish those updates and tap "Check for updates", it immediately shows few more app updates (3 apps in the screenshot.) After I finish those updates and tap "Check for updates" again, no more app updates show. 

In summary, notification said 23 app updates pending, but I got only 5 updates. 

This has started happening only after I used the  zygisk-detach module. I have detached only 3 apps using it -

- YouTube
- YouTube Music
- Instagram

If I reset the detachment, uninstall the module and reboot the phone, next time I receive the play store notification, the number of apps mentioned in the notification matches with the pending app updates shown in the updates section. 

That's why I'm thinking this discrepancy is caused by the module. But since it's not generating the logs either, not sure what other info can I provide to help find the cause!

**Comment by bagarwa** [comment]
The issue persists even after flashing the latest v1.12.0, reverting and then redoing the detachments.

**Comment by j-hc** [comment]
this isnt an actual issue. there is a discrepancy in notifications but it does not detach anything other than you selected.
notification issue will be fixed in v1.13.0
